README.md

```diff
@@ -8,6 +8,10 @@ production environment set the environmental variable `JEKYLL_ENV=production`.
 To watch for file changes and trigger a rebuild, you can run
 `jekyll build --watch`.
 
+For local environments you need to add the `_config_development.yml` file to the
+build to override the `baseurl`, by adding `--config _config.yml,_config_development.yml` to

```

Comment:
What `baseurl` is supposed to contain in a development environment ? `/` ? Might be worth documenting this.

Reply:
I added the `_config_development.yml` file, it's an empty string